frotz.py

```diff
@@ -50,11 +50,11 @@
     pass # FIXME
 else:
     if platform.architecture()[0] == '64bit':
-        vte_path = "x86-64"
+        vte_path = "x86_64"
     else:
         vte_path = "x86"
-    vte = cdll.LoadLibrary("lib/%s/libvte.so.9" % vte_path)
-    sys.path.append("lib/%s" % vte_path)
+    vte = cdll.LoadLibrary("/usr/lib/"+vte_path+"-linux-gnu/libvte-2.91.so.0")
+    sys.path.append("/usr/lib/"+vte_path+"-linux-gnu" )
```

Comment:
Is there any reason why you added this to the `PATH`?

Reply:
The activity was giving error before given in #4 in python2. The error is gone after changing . You can try this one also. by running on sugar-live-build. Thanks

Reply:
The activity was written to work with early versions of Fedora that did not have up to date VTE package.  Newer VTE was in the `lib` directory, for each architecture supported.  It also happened in the Lemonade activity.  This method was acceptable at the time but was not compliant with the license of VTE, and we no longer need to do it because VTE is sufficiently recent on any system that has Python 3 GTK support.  So as you port to GTK 3, you should have assessed if the embedded copy of VTE was needed, and (as it is not) removed it and the code that refers to it.

Reply:
And of course remove the `lib` directory.

Reply:
Thanks I didn't know that. I thought the error was just related to not loading the required lib. So I just made necessary changes for working. I will take care about it further.